auth-web/src/components/auth/account-settings/team-management/UserManagement.vue

```diff
@@ -310,19 +334,23 @@ export default class UserManagement extends Mixins(AccountChangeMixin, TeamManag
 
   private showConfirmRemoveInviteModal (invitation: Invitation) {
     this.confirmActionTitle = this.$t('confirmRemoveInviteTitle').toString()
-    this.confirmActionText = `Are you sure wish to remove the invite to ${invitation.recipientEmail}?`
+    this.confirmActionText = `Remove the invitation to sent to <strong>${invitation.recipientEmail}</strong>?`
```

Comment:
This text a bit confusing for me. may be just for me.

probably check with others too. @sumesh-aot @saravanpa-aot

Reply:
I've ran this through the UX team for feedback.  Fine as is.